Packs/GoogleChronicleBackstory/Playbooks/playbook-ChronicleAsset_Investigation_And_Remediation_-_Chronicle.yml

```diff
@@ -0,0 +1,764 @@
+id: ChronicleAsset Investigation And Remediation - Chronicle
```

Comment:
Look at previous comment and also - i don't see remediation actions in this sub-playbook, why does the playbook name contain the word remediation?

Reply:
@altmannyarden From what we have understood, a playbook can in a way be considered as a function.
In this case, according to our understanding since this playbook consists of a sub-playbook that performs the remediation, we recommend having "Remediation" in the name of this playbook. Please assist us with this, if we still need to change the name.

Reply:
The playbook name should be considered as the functions in that playbook, if the playbook doesn't contain any remediation actions - the name of the playbook shouldn't contain the name remediation (even if it's a sub-playbook)

Reply:
@altmannyarden Thank you so much for the explanation. We have applied these changes in [this ](https://github.com/demisto/content/pull/10022/commits/16ffc1758735f7ac94990651419a94961c81f329)commit.